Branch-TestBed/Branch-TestBed/AppDelegate.m

```diff
@@ -41,15 +41,23 @@ - (BOOL)application:(UIApplication *)application
     // test pre init support
     //[self testDispatchToIsolationQueue:branch]
     
-    
-    [Branch enableLoggingAtLevel:BranchLogLevelVerbose withCallback:^(NSString * _Nonnull message, BranchLogLevel logLevel, NSError * _Nullable error) {
+    [Branch enableLoggingAtLevel:BranchLogLevelVerbose withCallback:^(NSString * _Nonnull message, BranchLogLevel logLevel, NSError * _Nullable error, NSMutableURLRequest * _Nullable request, BNCServerResponse * _Nullable response) {
         // Handle the log message and error here. For example, printing to the console:
         if (error) {
             NSLog(@"[BranchLog] Level: %lu, Message: %@, Error: %@", (unsigned long)logLevel, message, error.localizedDescription);
         } else {
             NSLog(@"[BranchLog] Level: %lu, Message: %@", (unsigned long)logLevel, message);
         }
         
+        if (request) {
+            NSString *jsonString = [[NSString alloc] initWithData:request.HTTPBody encoding:NSUTF8StringEncoding];
```

Comment:
this gets rid of the class logging?

Reply:
The class details are still included in the log message. This part of the code is just part testbed app using the new request object.